### PR TITLE
Migrate txn_coord_sender_test.go from client.KV to TxnCoordSender.

### PR DESCRIPTION
**Pull request by petermattis**
See #997.

### CODE REVIEW

File: kv/txn_coord_sender_test.go [hunk 1]
```diff
@@ -443,30 +455,31 @@ func TestTxnCoordSenderCleanupOnAborted(t *testing.T) {
...
 
 	txn := newTxn(s.Clock, proto.Key("a"))
-	if err := kv.Run(client.Call{
+	call := client.Call{
 		Args:  createPutRequest(proto.Key("a"), []byte("value"), txn),
-		Reply: &proto.PutResponse{}}); err != nil {
+		Reply: &proto.PutResponse{},
+	}
+	s.Sender.Send(context.TODO(), call)
+	if err := call.Reply.Header().GoError(); err != nil {
```
Comment (vivekmenezes) [inline kv/txn_coord_sender_test.go]: I see this pattern repeat itself quite often in this file. How about creating a method that calls out like:

if err := SendPut(createPutRequest(proto.Key("a", []byte("value"), txn)); err != nil {

**Comment by tschottdorf** [comment]
LGTM

File: kv/txn_coord_sender_test.go (hunk 1 above)
Comment (tamird) [inline kv/txn_coord_sender_test.go]: +1

Comment (petermattis) [inline kv/txn_coord_sender_test.go]: I cleaned this up a bit by adding a `sendCall` helper. I'm sure there is much more that could be done to simplify this code, but at least the change is a bit more minimal now.

**Pull request merged**